modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationReply.java

```diff
@@ -144,18 +196,62 @@ private SyncmlBody generateStatuses() {
     private void appendOperations(SyncmlBody syncmlBody) throws WindowsOperationException {
         Get getElement = new Get();
         List<Item> itemsGet = new ArrayList<Item>();
-
-        Exec execElement = new Exec();
-        List<Item> itemsExec = new ArrayList<Item>();
-
+        List<Exec> execList = new ArrayList<>();
         Atomic atomicElement = new Atomic();
         List<Add> addsAtomic = new ArrayList<Add>();
 
+
         if (operations != null) {
             for (int x = 0; x < operations.size(); x++) {
                 Operation operation = operations.get(x);
                 Operation.Type type = operation.getType();
+
                 switch (type) {
+                    case POLICY:
+                        if (operation.getCode().equals("POLICY_BUNDLE"))
+                        {
```

Comment:
Formatting issues. Square bracket should be there right next to the closing bracket of the if-clause.